lib/hmmlearn/_hmmc.pyx

```diff
@@ -23,6 +23,8 @@ cdef inline dtype_t _max(dtype_t[:] X) nogil:
 
 cdef inline dtype_t _logsumexp(dtype_t[:] X) nogil:
     cdef dtype_t X_max = _max(X)
+    cdef ssize_t i
```

Comment:
I ain't a reviewer, just spotted this in the first line. Do you ever use `i` later on?